README.md

```diff
@@ -210,6 +210,12 @@ To configure the package behaviour, you can modify your `network_params.json` fi
         "tx_spammer_extra_args": []
     },
 
```

Comment:
somewhere up / down there's a `launch_additional_services` arg; can we say how one can add `gloomy_blob` there